hedera-node/src/test/java/com/hedera/services/state/merkle/MerkleAccountStateTest.java

```diff
@@ -135,7 +148,8 @@ public void release070DeserializeWorks() throws IOException {
 		given(in.readLong())
 				.willReturn(expiry)
 				.willReturn(balance)
-				.willReturn(autoRenewSecs);
+				.willReturn(autoRenewSecs)
+				.willReturn(nftsOwned);
```

Comment:
for release 070 we do not need to mock nftsOwned. They will not be deserialised because of the `if`